File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--programming-against-events-of-a-namedrange-control_1.vb
    Private Sub NamedRange1_BeforeDoubleClick( _
        ByVal Target As Microsoft.Office.Interop.Excel.Range, ByRef Cancel As Boolean) _
        Handles NamedRange1.BeforeDoubleClick

        With Me.NamedRange2
            .Value2 = "The BeforeDoubleClick event occurred."
            .Font.Color = System.Drawing.ColorTranslator.ToOle(System.Drawing.Color.Red)
            .Font.Italic = True
        End With
    End Sub

File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--programming-against-events-of-a-namedrange-control_1.cs
        private void namedRange1_BeforeDoubleClick(
            Microsoft.Office.Interop.Excel.Range Target, ref bool Cancel)
        {
            this.namedRange2.Value2 = "The BeforeDoubleClick event occurred.";
            this.namedRange2.Font.Color = System.Drawing.ColorTranslator.ToOle(System.Drawing.Color.Red);
            this.namedRange2.Font.Italic = true;
        }